### PR TITLE
readme added Admistrator rights description

### PR DESCRIPTION
**Pull request by earl-designs**
added the Admistrator rights description to the Readme for issue #1112

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -90,5 +90,32 @@ public void Monitor()
 }
 ```
 
+**Admistrator rights**
```
Comment (PhyxionNL) [inline README.md:93]: Administrator.

File: README.md [hunk 2]
```diff
@@ -90,5 +90,32 @@ public void Monitor()
...
+
+Some sensors require administrator privileges to access the data.
+
+Restart your IDE with admin privileges or add the following to your application to force admin rights on run
+
+#### **`app.manifest`**
```
Comment (PhyxionNL) [inline README.md:99]: Let's link to MSDN on how to add app.manifest to avoid poluting the readme.

File: README.md (hunk 1 above)
Comment (earl-designs) [reply README.md]: ups thank you ^-^ is corrected

File: README.md (hunk 2 above)
Comment (earl-designs) [reply README.md]: would this be ok?

**Pull request merged**